_posts/2021-03-29-Arts and Sciences Recipe Page.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+Author: Levi Blinder
+Title: Arts and Sciences Recipe Page
+---
+
+```
+<doc>
+...
+...
+...
+<div type = "recipe">
+  ...
+  ...
+  ...
+  <page>44</page>
+  <pageNum>14</pageNum>
+  <p>
+    it down with a <tl> String </tl> as far as
+    the water, then fire the <m> Gun Powder </m>
+    with a <tl> hot Tool </tl>, + it will clean the
+    well of every noxious vapor.
+  </p>
+</div>
+
+<red> <dbl> </dbl> </red>
+
+<div type = "recipe">
+  <recNum> <red> 16 </red> </recNum>
+  <head> <red> To Kill Buggs </red> </head>
+  <line> </line>
+  <p>
+    Mix <m> Oil of Turpentine </m> with
+    the <m> <pl> Neopolitan </pl> Ointment </m>, + smear
+    the Bedstead, + peel into the <q> ottebes </q>
+    where the Buggs are + they will be
+    destroyed.
+  </p>
+</div>
+
+<red> <dbl> </dbl> </red>
+
+<div type = "recipe">
+  <recNum> <brown> 17 </brown> </recNum>
+  <margin>
+    <brown> To <person> <q> Colo le tterine </q>
+    of the Artilery
+    at <pl> Gibralter </pl> </person>
+    <q> [Goent OZEH]</q> </brown>
+  </margin>
+  <head> <brown> Pigment for Wood </brown> </head>
+  <dbl></dbl>
+  <p>
+    <abr type = "recipe"> Rx </abr> <m> Pitch <qt> 10 tt </qt> </m> + <m> Tar <qt> tty </qt> </m> as much
+    <margin>
+      <m> Rosen <qt> tty+ </qt> </m>
+      <m> <qt> 1 Quart </qt> <q> Fabuin </q> Oil </m>
+    </margin>
+    <m> Grease or oil </m> as will give it the
+    Consistence of Paint, melt these together
+  </p>
+  <page> 44 </page>
+  ...
+  ...
+  ...
+</div>
+...
+...
+...
+</doc>
+```
+### What aspects of the document do I want to tag for a researcher?
+
+I wanted to capture aspects of the writing itself -- what role a word or phrase played in the recipe -- but also aspects of the physical  presentation of the text.
+
+The physical aspects that stood out to me were:
+* The use of various colors in different parts of the text
+* Lines and double lines used to segment the text
+* The presence of possibly illegible or unintelligible words/phrases
+* Writing that appears in the margins
+
+For the most part, I don't have a clear idea about what a researcher would do with these tags, but they were the physical features that struck me looking when looking at the document. I imagine that a researcher who isn't using the image of the document might need these tags to understand its layout and points of emphasis.
+
+I also do have a few vague ideas about how these could be used:
+* First, it seems possible that a researcher may only be interested in how a writer/reader used margin notes, in which case tagging these margin notes would allow the researcher to easily compile those notes.
+* Second, a researcher may be looking for phrases or words that are now obsolete and unrecognizable, in which case they might want to look through the words/phrases marked as illegible/unintelligible.  
```

Comment:
This is a really key insight--one of the beauties of XML when working with hard to decipher documents is that you can make visible what you _don't_ know as a researcher by tagging it for others to see.